### DISCUSSION TITLE
new package suggestions?

### DISCUSSION THREAD

**Discussion by lalberts** [chocolatey/home#180] [Q&A] [answered]
hello,
is there a way to encourage the presence of new packages in the choco list.
Probably one has to find a package maintainer for it?
But a suggestion list somewhere would be nice:
radiance-online.org
energyplus.net

for instance..
thanks

**Comment by pauby** [comment, marked as answer]
I'm not entirely sure I understand your suggestion. There is an existing [GitHub repository to request a new package or a new maintainer for an existing package](https://github.com/chocolatey-community/chocolatey-package-requests/issues).
